Sources/Workflow/LinkedList/LinkedListComparable.swift

```diff
@@ -13,12 +13,14 @@ extension LinkedList where Value: Comparable {
     /// Sorts the linked list in place using a merge sort.
```

Comment:
Isn't this a lie now, and you are not sorting in place?  If I have a reference to a node, and I call sort on the linked list, my node is not sorted, but my list is.  All references to my nodes are outdated at this point, right?

It could more realistically call out that it mutates the linked list into a sorted list, with an important note that previous references to nodes are now outdated.

Reply:
Perhaps, it's not a lie that it's sorted in place, because the *LinkedList* still is. But I agree I need to call out any references to nodes won't be changing. Lemme freak out over docs on that for a bit.